packages/lesswrong/lib/editor/ellipsize.tsx

```diff
@@ -30,6 +36,7 @@ export const truncate = (html: string|null|undefined, truncateLength: number, tr
     TruncateLength: Math.floor(truncateLength - (truncateLength/4)) || truncateLength,
```

Comment:
I'm pretty suspicious of this line of code, but it's been here for for 4 years so 🤷‍♂️

Reply:
Honestly I'm so suss of this line, I want to remove it unless someone can tell me a good reason. 🏈 to you to ask in slack? Or I can.